recipes/feets/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "feets" %}
+{% set version = "0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f74e539a385a182e9bddcc444c96639dd953d9cd04bd6865458817285e1fb207"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - astropy
+    - attrs
+    - mock
+    - numpy
+    - pandas
+    - pytest
```

Comment:
Do you need ``pyteste`` to run this project or just for testing purposes?

Reply:
and also ``mock``

Reply:
The test are integrated inside feets. Something like numpy 

```python
import numpy as np
np.test()
```

you can also run

```python
from feets import tests                                                                                                                                                                                    
tests.run()    
```

Reply:
I left `pytest` and `mock` in as dependencies since the desire is to always have the tests runnable from the package. Both are listed as dependencies in setup.py and are installed in `feets` is installed with `pip`.